documentation/reference/search-definitions-reference.html

```diff
@@ -2303,19 +2322,66 @@ <h2 id="field_types">Field types</h2>
 
 <tr><th id="type:array">array&lt;element-type&gt;</th>
 <td>
-Use to create a multi-value field of the element type.
-The element type can be any single value type, but not a nested array or weighted set.
+Use to create an array field of the element type.
+The element type can be of any single value type.
 <table class="table">
   <thead></thead><tbody>
     <tr>
       <th style="width:100px">Indexing</th>
       <td>Each element is indexed separately.</td>
     </tr><tr>
       <th>Attribute</th>
-      <td>Added as a multi-value attribute.</td>
+      <td>Added as an array attribute.</td>
     </tr><tr>
       <th>Summary</th>
-      <td>Added as a multi-value summary field.</td>
+      <td>Added as an array summary field.</td>
+    </tr>
+  </tbody>
+</table>
+</td></tr>
+
+<tr><th id="type:array-struct">array&lt;struct-type&gt;</th>
+<td>
+Use to create an array field of the given <a href="#struct">struct type</a>.
+The struct type must be defined separately.
+Restrictions:
```

Comment:
Should we move this _below_ the example to avoid breaking the flow of "here's what you do" and the example showing it? This also improves the proximity of "See table below for supported cases", since the table will be just below the list.

Also consider adding a `<p></p>` around Restrictions to ensure it gets a line of its own to strut around on (unless this looks like crap inside a table).